### ISSUE TITLE
RSLHelper unable to find Raid.exe process

### ISSUE THREAD

**Issue by Drauku** [FarbstoffRSL/RSL-Helper#414]
### Operating system

Windows 10

### Systemlanguage

English (EN)

### Current Game-Version

5.60.2

### RSL Helper Version

v3.2 Release-4 Build-2

### Bug description

I have been experiencing this bug for many months, and the only fix I have found so far is a system reboot.

Whenever I close the game, the Raid.exe process does not actually quit. I'm able to reload the game, but then there are multiple Raid.exe processes in task manager. It cannot be force closed, even via cmd line taskkill or using Process Hacker. The only fix is an OS reboot.

The issue this causes is RSLHelper apparently cannot differentiate between the old and new processes, and therefore can never attach to the new Raid process.

Note the two Raid.exe processes in the attached image, the current/new process 31728 is using RAM, the 27816 process is not, and is the dead/remnant old Raid.exe process:
![Ya4PGs54Ex](https://user-images.githubusercontent.com/17952866/174496821-353d4ba5-8501-49fa-8af4-e22183c2e2b1.png)
.

### Steps to reproduce

- Start Raid using RSLHelper. 
- Close Raid and RSLHelper. 
- Start Raid again using the RSLHelper app. This does load a new Raid instance as expected, however, the RSLHelper client now is stuck on "Wait for a new Raid Window" despite having opened the new window itself.

**Comment by Drauku** [comment]
The issue I'm reporting is that RSLHelper cannot differentiate between the stale raid.exe process and the new raid.exe process that was started by the helper itself.
I wouldn't know how to program for that, but I know it's possible since Plarium Play and the HH Gaming Artifact Extractor are both able to find the new process.

**Comment by Drauku** [comment]
Is this issue not something others have encountered? I've been seeing this behavior for many months, on three different Windows 10 desktops as well.

**Comment by OpaWattebausch** [comment]
No, you are the first and only one. Is your helper located in the standard path? If not: Did you set the path to plariumplay.exe in the settings?

**Comment by Drauku** [comment]
Standard path? Probably not, this is my installation path:
`C:\Games\RSL\RSLHelper`

Yes, my PlariumPlay.exe location is properly configured in settings. RSL Helper successfully loads Raid when I open it.

Are errors like this image possibly related? Note the rather large number in the CalcTime: filed, bottom right.
![image](https://user-images.githubusercontent.com/17952866/174727645-e9d48d53-fe35-4966-b0b6-173d106c78a9.png)

**Comment by OpaWattebausch** [comment]
Reinstall the vc++ redist package and don't forget the restart of pc after that.

**Comment by Sanych** [comment]
Попробуйте заново прописать путь к ярлыку Raid.exe в настройках RSL и поставить галочку на разрешение автоматического запуска
![баг](https://user-images.githubusercontent.com/108149870/175644355-1f7cbb2c-325a-45ac-a845-f2881e69c3ea.jpg)

**Comment by Drauku** [comment]
Ever since updating beyond v3.2 release 2 build 4, RSLHelper has not crashed with the "undefined operation" error as shown in my last post, just above.

I went ahead and reinstalled all vc++ redist packages from 2005-2019 just to make sure I did not leave one missing.

Per Sanych's suggestion, I went ahead and re-registered the PlariumPlay path, but loading Raid was never an issue.

I realize there is nothing that can be done about removing the remnant raid.exe process, that is a Plarium issue. I was just pointing out that other programs which also connect to raid processes do not run into this conflict with remnant processes. I do not know what they do differently, but it is possible for RSLHelper to continue working even without raid.exe exiting cleanly.
